sensors/omron_d6t/omron_d6t.py

```diff
@@ -0,0 +1,96 @@
+import time, crcmod.predefined
+from sensors.sensor import Sensor
+
+class OmronD6T(Sensor):
+    __START_COMMAND = 0x4c
+    __ARRAY_SIZE = 16
+    __BUFFER_LENGTH = 35
+    __PEC_BUFFER = [0x15]
+    __MAX_CELLS_ABOVE_ROOM_TEMPERATURE = 10
+
+    def __init__(self, bus = 1, address = 0x0a):
+        super().__init__(bus, address, self.__START_COMMAND)
+
+    @property
+    def buffer_length(self):
+        return self.__BUFFER_LENGTH
```

Comment:
Not sure properties are appropriate for constants. The whole point of the property is to act like a getter but that's only necessary to protect internal attributes that are variable and could be changed from the outside. Although constants are not technically immutable, they are by convention and thus don't need getters/property.